webapp/components/wpt-amend-metadata.js

```diff
@@ -59,7 +61,16 @@ class AmendMetadata extends LoadingState(ProductInfo(PolymerElement)) {
             <paper-input label="Bug URL" value="{{node.url}}" autofocus></paper-input>
           </div>
           <template is="dom-repeat" items="[[node.tests]]" as="test">
-            <li>[[test]]</li>
+            <template is="dom-if" if="[[hasHref(node.product)]]">
+              <li>
+                <a href="[[getSearchURLHref(test)]]" target="_blank">[[test]]</a>
+                <paper-tooltip offset="0" animation-delay="0">Search this test in Monorail</paper-tooltip>
```

Comment:
BTW, this should probably be inlined instead of in a tooltip. It's confusing what this linkified path is when you are just looking at the popup.

Reply:
How about a search-ish icon that has tooltip? Some test names tend to be quite long so I don't want the line even longer

Reply:
You could use `text-overflow: ellipsis` https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow to make the line doesn't get too lon.

Reply:
Thanks! I set the word limit to 100 characters~